app/config.js

```diff
@@ -135,14 +54,20 @@ exports.getPlugins = function () {
   };
 };
 
-exports.window = {
-  get() {
-    const position = winCfg.get('windowPosition');
-    const size = winCfg.get('windowSize');
-    return {position, size};
-  },
-  recordState(win) {
-    winCfg.set('windowPosition', win.getPosition());
-    winCfg.set('windowSize', win.getSize());
+exports.getKeymaps = function () {
+  return cfg.keymaps;
+};
+
+exports.extendKeymaps = function (keymaps) {
+  if (keymaps) {
+    cfg.keymaps = keymaps;
```

Comment:
Hmm.. I don't like mutating stuff like this. But I think we do this in other places? Maybe that's a job for another PR.